src/rpc/request.h

```diff
@@ -11,6 +11,11 @@
 
 #include <univalue.h>
 
+enum class JSONVersion {
```

Comment:
Is JSON/JSONRPC really necessary for the values themselves if the `enum` is of the type `enum class` so that one always has to prepend the type? - `JSONRPCVersion::JSONRPC_1_BTC` vs `JSONRPCVersion::1_BTC`.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/27101#discussion_r1594504094

> Is JSON/JSONRPC really necessary for the values themselves if the `enum` is of the type `enum class` so that one always has to prepend the type? - `JSONRPCVersion::JSONRPC_1_BTC` vs `JSONRPCVersion::1_BTC`.

Good point. I'd probably call the constants `V1_LEGACY` and `V2` if renaming.